jsons/1.12.2.json

```diff
@@ -719,6 +719,6 @@
     "minecraftArguments": "--username ${auth_player_name} --version ${version_name} --gameDir ${game_directory} --assetsDir ${assets_root} --assetIndex ${assets_index_name} --uuid ${auth_uuid} --accessToken ${auth_access_token} --userType ${user_type} --versionType ${version_type}",
     "minimumLauncherVersion": 18,
     "releaseTime": "2017-09-18T08:39:46+00:00",
-    "time": "2017-09-18T08:41:11+00:00",
+    "time": "2017-12-07T09:55:13+00:00",
```

Comment:
This file should not be part of your PR.

Reply:
So create a new PR?